packages/experiments/src/components/CollapsibleSection/CollapsibleSection.view.tsx

```diff
@@ -23,16 +24,71 @@ export const CollapsibleSectionView = (
 ) => {
   const { collapsed, titleAs: TitleType = CollapsibleSectionTitle, titleProps, children } = props;
 
-  // TODO: make sure onToggleCollapse gets called both from state and from titleProps.
+  const onRootKeyDown = (ev: React.KeyboardEvent<Element>) => {
```

Comment:
I'm not sure this belongs in the view. Shouldn't this be managed in a state component?

Reply:
Yeah... I had a comment about this on the PR.